index.html

```diff
@@ -53,13 +53,16 @@ <h1>CODEVID-19</h1>
       </p>
       <ul class="actions">
         <li>
-          <a href="join-the-fight" class="button">Join the Fight</a>
+          <a href="{{ '/join-the-fight' | relative_url }}" class="button primary">Join the Fight</a>
```

Comment:
Let's not change this link title in this PR as I think that will open a can of worms and keep this from going out.

Reply:
Before I do anything, I wanted to make sure my understanding is correct here.  I kept the title the same: did you mean the button style?  I can revert that no problem.

Where I'm becoming confused is when to use `relative_url`.  To me, I understand its benefits (if you use it in a `_layout/` file, it works for all pages), but maybe I should revert the use of it in this specific context.  I don't think we're going to be moving the landing page into a folder during the lifespan of this hackathon.  Overall I feel I was adding this to be "safe" from an unlikely scenario.

Reply:
Hey, to address the first comment I'm specifically talking about the link title. I realize it wasn't changed but changing it was mentioned so I wanted to way in.

Reply:
Oh okay.  I'll leave this as-is then.